utam-compiler/src/main/java/utam/compiler/translator/DefaultTranslatorRunner.java

```diff
@@ -254,6 +281,29 @@ public void writeDependenciesConfigs() {
     output.writeConfigToFile(configPath);
   }
 
+  /**
+   * build string with manifest path
+   *
+   * @return full path
+   */
+  final String buildManifestPath() {
+    String resourcesRootPath = getTargetConfig().getInjectionConfigRootFilePath();
+    if (resourcesRootPath == null || resourcesRootPath.isEmpty()) {
+      throw new UtamRunnerError(ERR_PROFILE_PATH_NOT_CONFIGURED);
```

Comment:
are we sure we want to throw profile error? seems like different use case

Reply:
Good catch. Fixed in subsequent commit.